### PR TITLE
Create Aladdin Alert

### PR DESCRIPTION
**Pull request by zsarnett**

### CODE REVIEW

File: alerts/aladdin_connect.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "Aladdin Connect is turning off their previous connection method"
+created: 2022-07-14 06:00:00
+integrations:
+  - yeelight
+homeassistant: "<2022.7"
+---
+
+## Summary
+
+Genie Company, owner of Aladdin products, is shutting down the previous method of connection that utilized scraping of the Android application. Home Assistant moved to the new connection using WebSockets in the 2022.7 release. If you are using a previous version then Home Assistant will no longer be able to connect to your Aladdin devices.
```
Comment (balloob) [inline alerts/aladdin_connect.markdown:11]:
```suggestion
Home Assistant 2022.6 and earlier used an old way of interacting with Aladdin products which is no longer supported.

Home Assistant 2022.7 implements a modern way. Users of Aladdin products should update as soon as possible to be able to continue using their Aladdin products uninterrupted.
```

**Comment by zsarnett** [comment]
Wait on the next hot fix of HA to publish

File: alerts/aladdin_connect.markdown [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Aladdin Connect is turning off their previous connection method"
+created: 2022-07-14 06:00:00
+integrations:
+  - yeelight
```
Comment (frenck) [inline alerts/aladdin_connect.markdown:5]:
```suggestion
  - aladdin_connect
```

**Comment by zsarnett** [comment]
😄 thanks frenck... missed that one

**Review by frenck — APPROVED** [review]

**Pull request merged**